### ISSUE TITLE
Custom training set appearance

### ISSUE THREAD

**Issue by bi94** [thtrieu/darkflow#1070]
Hi everyone,
first of all, since my question may seem a bit stupid, I apologize for that - I'm a neophyte to YoLo.

Let's go to the point: I have 9 GB videos recorded by an IR camera, and my task is to build a neural network capable of recognizing some details into them.

Starting from YoLo, I wrote the architecture of the ANN and successfully trained it on few labelled (with labelImg) screenshots of the videos: the problem is that, even if I give the net an already seen example, it can't recognize my custom objects. 

Where am I wrong? Labelling? Training set?

Many thanks,
bi94

**Comment by RaysirChang** [comment]
How about increasing the epoches & reducing the learning rate?

**Comment by Tanmay-Kulkarni101** [comment]
How many examples did you run it on? Have you made of the test, train and validation split, to check how well your model performs ?
